src/demo.js

```diff
@@ -88,9 +92,24 @@ export default class DemoTimeline extends Component {
       }
     }
 
+    const dataGridColumns = [
+      {
+        width: 100,
+        headerKey: 1
+      },
+      {
+        width: 250,
+        dataKey: 'description',
```

Comment:
Interesting this "default" way to render. I'd call it `labelProperty`. But we should also have the possibility to define easily a custom renderer. With `cellRenderer`.

I don't think it's a good idea to keep the default behavior of the group renderer, to consume from `title`. It's more explicit, when we have a table, to be forced to choose the name of each property.